.gitignore

```
@@ -12,5 +12,4 @@ bitrise-addon-service
 migrations/schema.sql
 .gows*
 *.css.map
-**/.env*
-vendor
\ No newline at end of file
+**/*.env*
```

What do you think about adding the bin/ folder to .gitignore to avoid binaries in git like the currently deleted files?
```suggestion
**/*.env*
bin/
```

---

Buffalo actually takes some time build, so I think it's a good idea to vendorize binaries in some cases. However, I didn't, because I was fighting with this build for so long I wanted to quickly reach a working solution